@stellar/typescript-wallet-sdk-km/webpack.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = (env = { NODE: false }) => ({
       stream: require.resolve("stream-browserify"),
       url: require.resolve("url"),
       util: require.resolve("util"),
+      vm: require.resolve("vm-browserify"),
```

Comment:
webpack 5 is throwing annoying warning saying we need to polyfill this now, so added